packages/esm-service-queues-app/src/queue-table/queue-table-cells.component.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { type VisitQueueEntry } from '../active-visits/active-visits-table.resource';
+import { ConfigurableLink, useConfig } from '@openmrs/esm-framework';
+import { type ConfigObject } from '../config-schema';
+
+export const QueueTableName = ({ queueEntry }: { queueEntry: VisitQueueEntry }) => {
+  const { customPatientChartUrl } = useConfig<ConfigObject>();
+  return (
+    <ConfigurableLink to={customPatientChartUrl} templateParams={{ patientUuid: queueEntry.queueEntry.patient.uuid }}>
+      {queueEntry.queueEntry.display}
+    </ConfigurableLink>
+  );
+};
+
+export const QueueTablePriority = ({ queueEntry }: { queueEntry: VisitQueueEntry }) => {
+  return <>{queueEntry.queueEntry.priority.display}</>;
```

Comment:
This component really should render a pill, as shown in the [designs](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/61bb8779a7fe1d4d8997a0fc).

Reply:
I'll make them pills. We might have to re-think the coloring if we want the flexibility of having available priorities configurable per queue (currently in [our spec](https://wiki.openmrs.org/pages/viewpage.action?pageId=351993893)). I'll worry about that in a later PR.

Reply:
Yeah, my original thought was to have the color based a `ConceptAttribute`, so the queue priorities could tell you how they should appear.